tests/e2e/cmux_test.go

```diff
@@ -157,10 +157,12 @@ func fetchGrpcGateway(endpoint string, httpVersion string, connType e2e.ClientCo
 func validateGrpcgatewayRangeReponse(respData []byte) error {
 	// Modified json annotation so ResponseHeader fields are stored in string.
 	type responseHeader struct {
+		//revive:disable:var-naming
 		ClusterId uint64 `json:"cluster_id,string,omitempty"`
 		MemberId  uint64 `json:"member_id,string,omitempty"`
```

Comment:
Ignored these as they come from protobufs and this is unmarshalling the response.